fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -3908,10 +3908,12 @@ class FileBackupAgentImpl {
 
 	ACTOR static Future<std::string> getStatusJSON(FileBackupAgent* backupAgent, Database cx, std::string tagName) {
 		state Reference<ReadYourWritesTransaction> tr(new ReadYourWritesTransaction(cx));
-		state JsonBuilderObject doc;
 
 		loop {
 			try {
+				state JsonBuilderObject doc;
+				doc.setKey("SchemaVersion", "1.0.0");
```

Comment:
Define a constant 'SchemaVersion' instead of using "1.0.0." in multiple places.

Reply:
The two document formats are independent, one is for description of data in a backup and the other is status about running backups.  They will evolve independently.